nodejs/eks/examples/tests/awsx-network-and-subnetIds/iam.ts

```diff
@@ -0,0 +1,27 @@
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+
+const managedPolicyArns: string[] = [
+    "arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy",
+    "arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy",
+    "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly",
+];
+
+// Creates a role and attaches the EKS worker node IAM managed policies
+export function createRole(name: string): aws.iam.Role {
```

Comment:
Does it make sense to include this as part of the EKS lib rather than defining in the test?

Reply:
We had the same question be asked by a user. Here's the thread:

- https://github.com/pulumi/pulumi-eks/pull/98#issuecomment-479978253
- https://github.com/pulumi/pulumi-eks/pull/98#issuecomment-480452516

In short:

> The approach here gives users who want to control the set of roles being used complete control over those roles and the associated instance profiles. It looks like the most flexible way to enable this while ensuring that all the related requests to this are now 

It's been mentioned to possibly embed this in `pulumi/awsx` in the past, but it never materialized. We've since copied around this `iam.ts` for the examples & tests that need them in `pulumi/eks` for node groups.